<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmyWidget</class>
 <widget class="QWidget" name="QmyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QmyWidget</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>578</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>设置男生年龄</string>
    </property>
    <property name="buddy">
     <cstring>spinBoxBoy</cstring>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxBoy">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>设置女生年龄</string>
    </property>
    <property name="buddy">
     <cstring>spinBoxGirl</cstring>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxGirl">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBoyInc">
    <property name="geometry">
     <rect>
      <x>272</x>
      <y>30</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>boy长大一岁</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClassInfo">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>108</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>类的元对象信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGirlInc">
    <property name="geometry">
     <rect>
      <x>274</x>
      <y>70</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>girl长大一岁</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClear">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>70</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>清空文本框</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>spinBoxBoy</zorder>
   <zorder>label_2</zorder>
   <zorder>spinBoxGirl</zorder>
   <zorder>btnBoyInc</zorder>
   <zorder>btnClassInfo</zorder>
   <zorder>btnGirlInc</zorder>
   <zorder>btnClear</zorder>
   <zorder>textBrowser</zorder>
  </widget>
  <widget class="QPlainTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>571</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
